src/openseadragon.js

```diff
@@ -518,7 +518,7 @@
   *     Milliseconds to wait after each tile retry if tileRetryMax is set.
   *
   * @property {Boolean} [useCanvas=true]
-  *     Deprecated. Use drawer option to specify preferred renderer.
+  *     Deprecated. Use options.drawer to specify preferred renderer.
```

Comment:
I think this is probably confusing as well... People probably don't think of the object they pass into the constructor as being called "options".

Maybe go back to the previous phrasing and put quotes or backticks around the word "drawer"?

```
Deprecated. Use the "drawer" option to specify preferred renderer.
```

Reply:
Done